client/landing/jetpack-cloud/sections/pricing/jpcom-masterbar/index.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * External dependencies
+ */
+import React, { useRef, useState } from 'react';
+import classNames from 'classnames';
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { Button } from '@automattic/components';
+import JetpackLogo from 'components/jetpack-logo';
+import useDetectWindowBoundary from 'my-sites/plans-v2/use-detect-window-boundary';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+const JETPACK_COM_BASE_URL = 'https://jetpack.com';
+const MENU_ITEMS = [
+	{
+		title: translate( 'Product Tour' ),
+		path: 'features',
+	},
+	{
+		title: translate( 'Pricing' ),
+		path: 'pricing',
+	},
+	{
+		title: translate( 'Support' ),
+		path: 'support',
+	},
+	{
+		title: translate( 'Blog' ),
+		path: 'blog',
+	},
+];
+
+const JetpackComMasterbar = () => {
+	const [ isMenuOpen, setIsMenuOpen ] = useState( false );
+	const barRef = useRef< HTMLDivElement | null >( null );
+	const hasCrossed = useDetectWindowBoundary( barRef );
+
+	const toggleMenu = () => {
+		setIsMenuOpen( ( currentState ) => ! currentState );
+	};
+
+	return (
+		<div ref={ barRef } className={ classNames( 'jpcom-masterbar', { sticky: hasCrossed } ) }>
+			<div className="jpcom-masterbar__inner">
+				<a
+					className="jpcom-masterbar__logo"
+					href={ JETPACK_COM_BASE_URL }
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					<JetpackLogo className="jpcom-masterbar__jetpack-logo" full size={ 48 } />
```

Comment:
I think we are off by a couple of pixels regarding the logos (this one vs jetpack.com's).

Reply:
You're right! I'll change it to 49.